WeakAuras/Prototypes.lua

```diff
@@ -1206,6 +1206,16 @@ function WeakAuras.GetEffectiveAttackPower()
   return base + pos + neg
 end
 
+--- @type fun(): number
+function WeakAuras.GetEffectiveSpellPower()
+  -- Straight from the PaperDoll
+  local spellPower = GetSpellBonusDamage(2)
+  for i = 3, MAX_SPELL_SCHOOLS do
+    spellPower = min(spellPower, GetSpellBonusDamage(i))
+  end
+  return spellPower
+end
```

Comment:
It doesn't look right to return minimal value, if i have 400 spell power and 600 fire spell power i expect it to return 600
Changed to look more similar to `WeakAuras.GetCritChance` and work on classic_era

```suggestion
--- @type fun(): number
function WeakAuras.GetEffectiveSpellPower()
  -- Straight from the PaperDoll
  local spellPower = 0
  for i = 2, MAX_SPELL_SCHOOLS or 7 do
    spellPower = max(spellPower, GetSpellBonusDamage(i))
  end
  return spellPower
end
```

Reply:
Hmm I checked the ui sources for what blizzard does, and to me the code looked like it did a min, though max should be much more useful in pratice. So yeah I'll change that.